bin/commands/transform.js

```diff
@@ -64,7 +66,7 @@ const promptOptions = [
   {
     ...fuzzyPathOptions,
     name: 'path',
-    message: 'Enter the path to service(s) file(s)/folder',
+    message: 'Enter the path to a file(s)/folder',
```

Comment:
I don't think plural is possible here, isn't it either one file or one folder?

Reply:
yeah you are right, you can pass multiple files to `jscodeshift` but with the fuzzy search plugin I don't think we pass multiple arguments.  updated.